docs/source/sctool/repair.rst

```diff
@@ -388,8 +388,8 @@ In addition to :ref:`global-flags`, repair update takes the same parameters as `
 repair update example
 .....................
 
-This example updates the repair task 143d160f-e53c-4890-a9e7-149561376cfd adding an intensity parameter to speed up the repair.
+This example updates the repair task 143d160f-e53c-4890-a9e7-149561376cfd, adding an intensity parameter to speed up the repair.
 
 .. code-block:: none
 
-   sctool repair update -c prod-cluster repair/143d160f-e53c-4890-a9e7-149561376cfd --intensity 0
```

Comment:
?

Reply:
Comma added line391 after letter d, if you feel it's not needed, that's ok.

Reply:
Comma is fine. There is another (void) change in 395.